### DISCUSSION TITLE
Filters are not working

### DISCUSSION THREAD

**Discussion by deepak458** [rappasoft/laravel-livewire-tables#986] [Q&A]
I have new with this package, and most of the working fine but all filters in top "search", "perpage", "Column" etc are not working.

any help ?

`    protected $model = Company::class;

    public function configure(): void
    {
        $this->setPrimaryKey('id');
        $this->setEmptyMessage('No results found');

        $this->setConfigurableAreas([
            'toolbar-left-end' => 'companies.create-company-btn',
        ]);
    }

    public function filters(): array{
        return [
            SelectFilter::make('Deleted')->options(['' => 'No', 'yes' => 'Yes'])
                ->filter(function(Builder $builder, string $value) {
                    if ($value === 'yes') {
                        $builder->onlyTrashed();
                    }
                })
        ];
    }

    public function columns(): array
    {
        return [
            Column::make("ID", "id")
                ->hideIf(true),
            Column::make("Name", "name")
                ->sortable()
                ->searchable(),
            Column::make("Phone", "phone")
                ->sortable()
                ->searchable(),
            Column::make("Email", "email")
                ->sortable()
                ->searchable(),
            Column::make("Status", "status")
                ->sortable(),
            ButtonGroupColumn::make('Actions')
                ->attributes(function($row) {
                    return [
                        'class' => 'space-x-2',
                    ];
                })
                ->buttons([
                    LinkColumn::make('View') // make() has no effect in this case but needs to be set anyway
                    ->title(fn($row) => 'View ')
                        ->location(fn($row) => route('companies.show', $row))
                        ->attributes(function($row) {
                            return [
                                'class' => 'underline text-blue-500 hover:no-underline',
                            ];
                        }),
                    LinkColumn::make('Edit')
                        ->title(fn($row) => 'Edit')
                        ->location(fn($row) => route('companies.edit', $row))
                        ->attributes(function($row) {
                            return [
                                'class' => 'underline text-blue-500 hover:no-underline',
                            ];
                        }),
                ]),
        ];
    }`

**Comment by lrljoe** [comment]
What happens if you remove your create button please, does it work as expected?
